airflow/migrations/versions/c2091a80ac70_create_scheduler_state_table.py

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""create scheduler_state table
+
+Revision ID: c2091a80ac70
+Revises: d2ae31099d61
+Create Date: 2019-11-14 16:22:10.159454
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'c2091a80ac70'
+down_revision = 'd2ae31099d61'
```

Comment:
When we pull in upstream changes, we'll just need to change this down_revision to be the last revision in the alembic versions folder.  It's adding an entirely new table, so unless upstream made a table with the same name this should not be a problem.  Incidentally making these is as simple as:

`alembic revision -m "create scheduler_state table"`

and then changing upgrade() and downgrade().  (I was pleasantly surprised by how simple it was.)